File: strings/6_Excel_sheet_column_Title.cpp
#define MAX 10
class Solution {
public:
    string convertToTitle(int columnNumber) {
       int n = columnNumber;
       char str[MAX]; 
       int i = 0; 
       while (n > 0) {
            int r = n % 26; // here r is remainder
            if (r == 0) {
                str[i++] = 'Z';
                n = (n / 26) - 1;
            }
            else {
                str[i++] = (r - 1) + 'A';
                n = n / 26;
            }
        }
        str[i] = '\0';
        reverse(str, str + strlen(str));
        // cout << str << endl;
 
    return str;
    }
};

File: Strings/6_Excel_Sheet_Column_Title.py
class Solution:
    def convertToTitle(self, columnNumber: int) -> str:
        title = "" #Creating an empty string to concate the title
        while columnNumber:
            columnNumber = columnNumber - 1 #As alphabets start from 0 we decrease the count by 1
            title = chr(columnNumber%26+65) + title #We get the number and covert it into the required character and concate
            columnNumber = columnNumber//26 #We floor the number to get remaining alphabets
        return title # At last we return the title